test/support/test_helpers.ex

```diff
@@ -0,0 +1,35 @@
+defmodule EspiDni.TestHelpers do
+  alias EspiDni.Repo
+
+  def insert_team(attrs \\ %{}) do
+    changes = Dict.merge(%{
```

Comment:
Should it use `Map`? `Dict` is deprecated: http://elixir-lang.org/docs/stable/elixir/Dict.html